charts/milvus-operator/Chart.yaml

```diff
@@ -18,7 +18,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.7.5
```

Comment:
Not sure if this is how you manage versions or not... couldn't find any instructions. Let me know if I need to change anything.

Reply:
We usually bump the version when we release a new formal version. This is done by running the script under test/bump-version.sh. 
It's ok to update it earlier, don't worry